### ISSUE TITLE
Preview fails to open due to hardcoded paths

### ISSUE THREAD

**Issue by martinpengellyphillips** [vscode-restructuredtext/vscode-restructuredtext#11]
Preview fails to open due to hard coded paths that expect docs in specific location:
https://github.com/vscode-restructuredtext/vscode-restructuredtext/blob/master/src/extension.ts#L123

Error is:

```
ENOENT: no such file or directory, stat 'c:\Users\Martin\data\identities\personal\project\lucidity\_build\html\doc\guide\introduction.html'
```

Actual files after build are located at `c:\Users\Martin\data\identities\personal\project\lucidity\build\doc\html\guide\introduction.html`

Note that target for built docs can be controlled through various means by project owner so might be worth letting that path be a configuration variable.

Additionally, the rest of the path should be relative to the `conf.py` I think rather than relative to the root of the workspace (in my example the docs are under a `doc` subdirectory, but that won't be in the generated html path).

**Comment by lextm** [comment]
Related to #10.

Yes, as you discovered, the current preview uses a hard coded path to locate the generated HTML pages.

The proper approach should be as you described, to locate conf.py first, parse it, and then know which path should go. 

Pull requests are welcome before I can allocate some time to fix it.

**Comment by martinpengellyphillips** [comment]
Cool - I'll try and work up a PR for this.

**Comment by martinpengellyphillips** [comment]
See PR #13 for proposed solution.

**Comment by lextm** [comment]
Closed via #13 

Thank you @martinpengellyphillips for the contribution.
